Simplify VEBA versioning to support automated Development branch builds (#234)
vmware-samples/vcenter-event-broker-appliance: docs/kb/contribute-appliance.md

> **Note:** If you need to change the default root password on the VMware Event Broker Appliance, take a look at `photon-version.json`

> Step 3 - The `veba-bom.json` will need to be updated to specify the branch you wish to build the vCenter Event Broker Appliance code from whether that is from master, a release- branch or from development. Below is an example of the three versions that needs to be updated prior to building.

Reviewer: ```suggestion
Step 3 - The `veba-bom.json` will need to be updated to specify the branch you wish to build the vCenter Event Broker Appliance code from whether that is from master, a release- branch or from development. Below are two examples of how to correctly set the versions needed prior to building.
```

> **Note:** The default BOM version in development will be the development branch. No changes will be necessary unless you wish to build from a release or master branch.

> ```

Reviewer: Can we change this to the following to avoid ambiguity (arrays)?

Example 1 (build from `master` branch):

```
".veba.version" => "v0.5.0"
"vmware-event-router.version" => "v0.5.0"
"vmware-event-router.containers[0].version" => "v0.5.0"
```

Example 2 (build from `development` branch):

```
".veba.version" => "development"
"vmware-event-router.version" => "development"
"vmware-event-router.containers[0].version" => "development"
```
